.changeset/famous-pets-notice.md

```diff
@@ -0,0 +1,8 @@
+---
+"@jpmorganchase/uitk-core": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Rename Drop Target characteristic to Target;
```

Comment:
I like how this changeset was done with renames https://github.com/jpmorganchase/uitk/pull/634/files#diff-84f7136ddb182f738b5cbdd44309e8b70e3e25af94c7913fd6b46b92acb9f627R6

I think it's worth copying it here